versioned_docs/version-v2.18.x/user-guide/zos-components-installation-checklist copy.md

```diff
@@ -0,0 +1,90 @@
+|----|-----------------------|---|
+| [Review the Zowe server-side installation overview](../user-guide/install-zos.md) | Knowledge about  the basic installation stages and the roles and responsibilities to perform the installation |    25 minutes   |                        
+| [Prepare for installation](./installandconfig)| Knowledge about the key-concepts in server-side installation |  25 minutes                    | 
+| Address pre-installation requirements | The following pre-installation requirements are addressed: <br /> &nbsp;&nbsp;&nbsp;&nbsp;* [z/OS](./systemrequirements-zos)   <br />&nbsp;&nbsp;&nbsp;&nbsp;* [Node.js](./install-nodejs-zos.md)   <br />&nbsp;&nbsp;&nbsp;&nbsp;* [security](./address-security-requirements) <br />&nbsp;&nbsp;&nbsp;&nbsp;* [USS](./configure-uss) <br />&nbsp;&nbsp;&nbsp;&nbsp;* [storage](./address-storage-requirements) <br />&nbsp;&nbsp;&nbsp;&nbsp;* [network](./address-network-requirements)  <br /> &nbsp;&nbsp;&nbsp;&nbsp;* [z/OSMF](../user-guide/systemrequirements-zos.md#zosmf-optional) <br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(recommended for full functionality)<br />&nbsp;&nbsp;&nbsp;&nbsp;* [z/OSMF HA](../user-guide/zowe-ha-overview.md)<br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(required for production)            |  1 day                     | 
+
+## Installing the Zowe z/OS runtime
+
+Choose from the following installation options to install Zowe server-side components for z/OS.
+
+| Task | Results | Time Estimate |  
+|--------------------|----|------|
+|  **Option 1: [Install Zowe with SMP/E](./install-zowe-smpe)** <br />(Optionally install via [Server Install Wizard](./install-zowe-server-install-wizard.md))<br /><br /> **Option 2: [Install Zowe with z/OSMF from a portable software instance](./install-zowe-pswi)** <br /><br /> **Option 3: [Install Zowe SMP/E build with z/OSMF workflow](./install-zowe-smpe-zosmf-workflow)** <br /><br /> **Option 4: [Install Zowe via a convenience build (PAX file)](../user-guide/install-zowe-zos-convenience-build.md)**<br />(Optionally install via [Server Install Wizard](./install-zowe-server-install-wizard.md))                     | Executables and binaries are unpaxed on the mainframe                |   1 hour                     |
+
+## Configuring Zowe z/OS Components 
+
+Choose the following options to initialize  Zowe z/OS runtime:
+
+| Task | Results | Time Estimate |  
+|-----------------------|----|------|
+|  **Option 1: [Configure Zowe with zwe init](../user-guide/initialize-zos-system.md)**  <br /><br /> **Option 2: [Configure Zowe with z/OSMF workflows](../user-guide/configure-zowe-zosmf-workflow.md)**  <br /><br /> **Option 3: [Configure Zowe with JCL](../user-guide/configuring-zowe-via-jcl.md)** | * All datasets are created and populated. <br /> * Started tasks are copied to system libraries.  <br /><br />**Important:** Security administrator permissions are required for some zwe init sub-commands to pass.   | 1 hour 
```

Comment:
We can also aadd the `via jcl` option in the `zos-components-installation-checklist.md` file